src/main/jbake/content/hawkular-metrics/docs/user-guide/index.adoc

```diff
@@ -1679,6 +1679,8 @@ then needs the prefix to uniquely identify the correct metric.
 
 |===
 
+Metrics has some special support for stream-based alerting, see <<Metrics Group Triggers>> for the details.
```

Comment:
I wonder whether it is better to place the MGT here, instead to link it and place it at the bottom.